crates/bonsaidb-core/src/admin/user.rs

```diff
@@ -25,12 +24,13 @@ pub struct User {
     /// The IDs of the roles this user has been assigned.
     pub roles: Vec<u64>,
 
-    /// An `OPAQUE PAKE` payload.
-    pub password_hash: Option<ServerFile>,
-
-    /// A temporary password state. Each call to SetPassword will overwrite the
-    /// previous state.
-    pub pending_password_change_state: Option<ServerRegistration>,
+    /// The user's stored password hash.
+    ///
+    /// This field is not feature gated to prevent losing stored passwords if
+    /// the `password-hashing` feature is disabled and then re-enabled and user
+    /// records are updated in the meantime.
+    #[serde(default)]
+    pub argon_hash: Option<String>,
```

Comment:
Maybe you want to wrap this in `ZeroizeOnDrop`.

Reply:
I'll rename Password to `SensitiveString`, changing it to `multiuser`, and reusing it here. Good call.